QUICK_START_GUIDE.md

```diff
@@ -0,0 +1,272 @@
+# Quick Start Guide
+
+## Prerequisites
+* Ubuntu 18.04 / 20.04
+* Python 3.8+
+* [OpenVINO™](https://software.intel.com/en-us/openvino-toolkit) - for exporting and running models
+* [CUDA Toolkit 11.1](https://developer.nvidia.com/cuda-11.1.1-download-archive) - for training on GPU
+
+## Setup OpenVINO™ Training Extensions
+
+1. Clone repository in the working directory by running the following:
+    ```
+    git clone https://github.com/openvinotoolkit/training_extensions.git
+    cd training_extensions
+    git checkout -b develop origin/develop
+    git submodule update --init --recursive
+    ```
+
+2. Install prerequisites by running the following:
+    ```
+    sudo apt-get install python3-pip python3-venv
+    ```
+
+3. Search for available scripts that create python virtual environments for different task types:
+   ```bash
+   find external/ -name init_venv.sh
+   ```
+
+   Approximate output:
+   ```
+   external/mmdetection/init_venv.sh
+   external/mmsegmentation/init_venv.sh
+   external/deep-object-reid/init_venv.sh
+   ```
+
+4. Let's create, activate Object Detection virtual environment and install `ote_cli`:
+   ```
+   ./external/mmdetection/init_venv.sh det_venv
+   source det_venv/bin/activate
+   pip3 install -e ote_cli/
+   ```
+
+## OTE CLI commands
+
+### ote find - search for model templates
+   Have a look at model templates available for this virtual environment:
+   ```
+   ote find --root ./external/mmdetection/
+   ```
+
+   Approximate output:
+   ```
+   - framework: OTEDetection_v2.9.1
+     name: Custom_Object_Detection_Gen3_VFNet
+     path: ./external/mmdetection/configs/ote/custom-object-detection/gen3_resnet50_VFNet/template.yaml
+     task_type: DETECTION
+   - framework: OTEDetection_v2.9.1
+     name: Custom_Object_Detection_Gen3_SSD
+     path: ./external/mmdetection/configs/ote/custom-object-detection/gen3_mobilenetV2_SSD/template.yaml
+     task_type: DETECTION
+   - framework: OTEDetection_v2.9.1
+     name: Custom_Object_Detection_Gen3_ATSS
+     path: ./external/mmdetection/configs/ote/custom-object-detection/gen3_mobilenetV2_ATSS/template.yaml
+     task_type: DETECTION
+   - ...
+   ```
+   Let's choose `./external/mmdetection/configs/ote/custom-object-detection/gen3_mobilenetV2_ATSS/template.yaml`
+
+### ote train - run training of particular model template
+   Let's have a look at `ote train` help. These parameters are the same for all model templates.
```

Comment:
Maybe it is worth pointing out, that to create the necessary files (`label_schema.json`) for scripts (`eval`, `export`) you need to first run `ote train`?

Reply:
added